src/components/ContentList/ContentListItem.vue

```diff
@@ -48,6 +45,16 @@ export default {
 			} catch (e) {
 				return 'grey'
 			}
+		},
+		avatarUrl() {
+			return 'url(' + this.contact.url + '?photo' + ')'
```

Comment:
Small tip, with latest javascript you can do
```js
`url(${this.contact.url}?photo)`
```

Reply:
Cool very nice, anything else here to change?

Reply:
I'll rebase your branch in a second :)